apps/web/src/env.ts

```diff
@@ -10,10 +10,12 @@ export default createEnv({
     NODE_ENV: z.string(),
     DATABASE_URL: z.string(),
     WEBSOCKETS_SERVER: z.string(),
+    DATASET_GENERATOR_WORKSPACE_APIKEY: z.string(),
```

Comment:
nein, declare it in packages/env please 🙏🏼 , I'm in the process of removing this env file